### DISCUSSION TITLE
Judging Tools

### DISCUSSION THREAD

**Discussion by mishmanners** [github/hackathons#4] [Q&A]
What are good sources for judging hackathons?

I only know of a couple, and a lot of hackathons I've been to even build their own judging tools. Aside from spreadsheets or Google forms, have you discovered a really good tool or platform for your judges to judge hackathon entries?

**Comment by devmike123** [comment]
You can also use TypeForm for receiving entries. To manage hackathon entries, check out this: [HackathonManager](https://github.com/codeRIT/hackathon-manager),[Quill](https://github.com/techx/quill),[HackAssistant](https://github.com/HackAssistant/registration).

Also I recommend to integrate Google form or TypeForm with Accredible to certify the participants or provide them a digital badge for participating in the hackathon.

@mishmanners

**Reply by devmike123** [reply]
You can check out this:
https://guide.mlh.io/organizer-resources/software-for-hackathons
